_v2_0_0-docs/installation.md

```diff
@@ -9,26 +9,227 @@ permalink: v2_0_0-docs/installation/
 
 ## General prerequisites
 
-* [Java 7 or later](http://www.oracle.com/technetwork/java/javase/downloads/index.html);
-* [Apache Cassandra 2.x.x](http://cassandra.apache.org/);
-* [Apache Solr 4.7.0](http://lucene.apache.org/solr/), Note that more recent versions of Solr are known to have an issue with the popularity index field;
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html);
+* [Elasticsearch 1.5.2](https://www.elastic.co/downloads/elasticsearch);
 * Credentials to our Nexus publishing repos.
 
 ## Standard (and easy) setup
 
 This guide is the bare minimum that needs to be done in order to setup a working system. Scalability and high availability are addressed separately.
 
-1. Install and run Cassandra.
-2. [Install Solr 4.7.0](https://cwiki.apache.org/confluence/display/solr/Installing+Solr).
+1. [Install Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/1.5/_installation.html).
+2. Create a `recommendation-reads` index using the following command containing the mappings and settings:
 
-	2.1. In your `solr.solr.home` directory create a new directory named `recommend` and unpack the contents of [`recommendation-solr-core-1.x.y-bin.zip`](http://maven.ontotext.com/content/repositories/publishing-releases/com/ontotext/recommend/recommendations-solr-core/1.2.0/recommendations-solr-core-1.2.0-bin.zip) there.
+<pre><code>
+curl -f -XPUT http://elastic-host:elastic-port/recommendations-reads -d '{
```

Comment:
If we use aliases, we should use another name for the index -  `recommendation-reads-1` or whatever convention we choose.